pynxtools/eln_mapper/README.md

```diff
@@ -1,10 +1,11 @@
 # ELN generator
-This is a helper tool for generating eln
-- The simple eln generator that can be used in a console or jupyter-notebook
-- Scheme based eln generator that can be used in NOMAD and the eln can be used as a custom scheme in NOMAD.
+This is a helper tool for generating ELN files that can be used to add metadata to the dataconverter routine.
+Two types of ELN are supported (by passing the flag `eln-type`):
+- **eln**: The simple ELN generator that can be used in a console or jupyter-notebook.
+- **scheme_eln**: Scheme based ELN generator that can be used in NOMAD and the ELN can be used as a custom scheme in NOMAD.
 
 ```
-$ eln_generator --options <value>
+$ generate_eln --options <value>
```

Comment:
In other places we just copy the output of `generate_eln --help`. I think we should follow this here, too.

Reply:
It currently looks like this (sorry, github did not let me make a multiline suggestion):

```
$ generate_eln --help
Usage: generate_eln [OPTIONS]

  To generate ELN in yaml file format.

  Parameters ----------

      nxdl : str         Name of NeXus definition e.g. NXmpes
      skip_top_levels : int         To skip hierarchical levels
      output_file : str         Name of the output file.

Options:
  --nxdl TEXT                  Name of NeXus definition without extension
                               (.nxdl.xml).  [required]
  --skip-top-levels INTEGER    To skip the level of parent hierarchy level.
                               E.g. for default 1 the partEntry[ENTRY] from
                               /Entry[ENTRY]/Instrument[INSTRUMENT]/... will
                               be skiped.  [default: 1]
  --output-file TEXT           Name of file that is neede to generated output
                               file.
  --eln-type [eln|scheme_eln]  [required]
  --help                       Show this message and exit.
```

Reply:
We should also remove the parameters part in the docs as this is automatically generated by click.

Reply:
Done!